configure.sh

```diff
@@ -17,57 +23,116 @@ PRIVATE_PUSHWOOSH_PROPERTIES="$BASE_PATH/android/pushwoosh.properties"
 PRIVATE_LIBNOTIFY_PROPERTIES="$BASE_PATH/android/libnotify.properties"
 PRIVATE_NETWORK_CONFIG="$BASE_PATH/android/res/xml/network_security_config.xml"
 SAVED_PRIVATE_REPO_FILE="$BASE_PATH/.private_repository_url"
+SAVED_PRIVATE_BRANCH_FILE="$BASE_PATH/.private_repository_branch"
 TMP_REPO_DIR="$BASE_PATH/.tmp.private.repo"
 
-# TODO: Remove these lines when XCode project is finally generated by CMake. 
-cd $BASE_PATH/3party/boost/
-./bootstrap.sh
-./b2 headers
-cd $BASE_PATH
+usage() {
+  echo "This tool configures omim with private repository or as opensource build"
+  echo "Usage:"
+  echo "  $0 private_repo_url [private_repo_branch]  - to configure with private repository"
+  echo "  echo '[private_repo_url] [private_repo_branch]' | $0  - alternate invocation for private repository configuration"
+  echo "  $0  - for opensource build or use of saved repository url and branch"
```

Comment:
to use with saved repository url and branch or to set up an open source build if nothing is saved

Reply:
Yes, changed to your variant.